File: programming/problem4.c
// Write a program in C to allow users to enter an arithmetic expression and display the result!



#include <stdio.h>

int main()
{
    int num1, num2;
    int sum, sub, mult, mod;
    float div;

    /*
     * Input two numbers from user
     */
    printf("Enter 1st number:");
    scanf("%d", &num1);
    printf("Enter 2nd number:")
    scanf("%d", &num2)

    /*
     * Perform all arithmetic operations
     */ 
    sum = num1 + num2;
    sub = num1 - num2;
    mult = num1 * num2;
    div = (float)num1 / num2;
    mod = num1 % num2;

    /*
     * Print result of all arithmetic operations
     */
    printf("SUM = %d\n", sum);
    printf("DIFFERENCE = %d\n", sub);
    printf("PRODUCT = %d\n", mult);
    printf("QUOTIENT = %f\n", div);
    printf("MODULUS = %d", mod);

    return 0; 
}

File: programming/problem4.rb
# Write a program in Ruby to allow users to enter an arithmetic expression and display the result!

print 'Enter a number:'
num1 = gets

print 'Enter another number:'
num2 = gets

puts 'Choose an operation:'
puts 'A) +'
puts 'B) -'
puts 'C) *'
puts 'D) /'

print 'Your choice:'
choice = gets

case choice
when 'A'
    puts "The result of your operation is #{num1.to_i+num2.to_i}"
when 'B'
    puts "The result of your operation is #{num1.to_i-num2.to_i}"
when 'C'
    puts "The result of your operation is #{num1.to_i*num2.to_i}"
when 'D'
    puts "The result of your operation is #{num1.to_i/num2.to_i}"
else 
    puts 'Please enter a valid choice'
end